Core/GDCore/IDE/Events/EventsPositionFinder.h

```diff
@@ -0,0 +1,50 @@
+#ifndef EventsPositionFinder_H
+#define EventsPositionFinder_H
+#include "GDCore/Events/EventsList.h"
+#include "GDCore/IDE/Events/ArbitraryEventsWorker.h"
+#include "GDCore/String.h"
+
+namespace gd {
+class BaseEvent;
+class EventsList;
+}  // namespace gd
+
+namespace gd {
+
+/**
+ * \brief Scans an event list to retrieve the position of a list of searched
+ * events when the events list is flattened.
+ *
+ * \ingroup IDE
+ */
+class GD_CORE_API EventsPositionFinder : public ArbitraryEventsWorker {
+ public:
+  EventsPositionFinder() {
+    index = 0;
```

Comment:
Uber nitpicking

```suggestion
  EventsPositionFinder() : index(0) {
```

The initializer list is recommended because it avoids to initialize to a garbage value then replace this value.

Reply:
Ok I see, thanks!